CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+Thank you for considering contributing to Minimal Commit! Your contributions are valuable in helping us improve and grow. To ensure a smooth and collaborative development process, please follow these guidelines when working on issues:
+
```

Comment:
Add a **Setup** section highlighting steps to set up the repo locally. Also, add **Prerequisites**, highlighting what's needed to work on the project (`Nodejs)`